package.json

```diff
@@ -27,7 +27,7 @@
     "internal:release:next": "yarn prerelease && yarn changeset publish --tag next",
     "serve": "cd ./site && next",
     "start": "npm-run-all --parallel --print-label watch serve",
-    "test": "mocha --require ./config/babel/register.cjs ./packages/*/test/index.js",
+    "test": "mocha --require ./config/babel/register.cjs ./packages/*/test/**/*.{js,ts}",
```

Comment:
is this relevant for this PR?

Reply:
Yes it is, without this change, tests in `test/utils/string.ts` do not run.